spec/status.yml

```diff
@@ -159,7 +159,7 @@ components:
       get:
         tags:
         - 'status'
-        operationId: "authenticatorStatus"
+        operationId: "authenticator"
         summary: "Details whether an authentication service has been configured properly"
         description: "Once the status webservice has been properly configured and the relevant user
```

Comment:
We should enumerate in the description the authenticators that support this route. They include:
- GCP Authenticator

We can also link to the [documentation](https://docs.conjur.org/Latest/en/Content/Integrations/Authn-status.htm) on configuring the Conjur server to enable certain users and groups to access this endpoint.